README.md

```diff
@@ -1 +1,237 @@
+
+- [ ] Once the files are on your machine, open the _blog-app_ folder in your code editor.
+Run the following command in your terminal to install the required gems and run the application:
+
+- [ ] Open the config/database.yml file in the project directory and change the username and password to your PostgreSQL username and password.
+Edit the default section of the file to look like this:
+    
+```
+  default: &default
+  adapter: postgresql
+  encoding: unicode
+  # For details on connection pooling, see Rails configuration guide
+  # http://guides.rubyonrails.org/configuring.html#database-pooling
+  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  username: <your PostgreSQL role username>
+  password: <your PostgreSQL role password>
+
+```
+
+### Install
+
+`NOTE:` _You may need to run this command in the project directory to install the required gems and run the application:_
+
+```
+bundle install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+1. Create the database with:
+
+```sh
+rails db:create
+```
+
+2. Run the migrations with:
+
+```sh
+rails db:migrate
+```
+
+3. Start the development server with:
+
+```sh
+rails server
+```
+
+4. Open the app in your browser at http://localhost:3000
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  bundle exec rspec
+```
+
+### Deployment
+
+You can deploy this project using:
+
+1. Deplay to Heroku
+
+```sh
+  git push heroku main
+```
+
+2. Or deploy to Render
+
+```sh
+  git push render main
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Author <a name="authors"></a>
+
+👤 **Desmond Owusu Ansah**
+
+- Github [@Owusu-Desmond](https://github.com/Owusu-Desmond)
+- Twitter [@DesmondOwusuDev](https://twitter.com/DesmondOwusuDev)
+- LinkedIn [@desmond-owusu-ansah](https://www.linkedin.com/in/desmond-owusu-ansah-09274a223/)
+- Dev.to [desmondowusudev](https://dev.to/desmondowusudev)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Add a feature to allow users to add a profile picture.]**
+- [ ] **[Add a feature to allow users to add a cover image]**
+- [ ] **[Add a feature to allow users to add a bio.]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/dodoburner/budget-app/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Write a message to encourage readers to support your project
+
+If you like this project...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
```

Comment:
- [ ] The README should contain an "Acknowledgements" section or similar that gives appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/).